code/game/turfs/turf.dm

```diff
@@ -323,3 +323,9 @@
 	SSair.remove_from_active(T0)
 	T0.CalculateAdjacentTurfs()
 	SSair.add_to_active(T0,1)
+
+/turf/proc/is_transition_turf()
+	if(istype(src, /turf/open/space))
```

Comment:
Have this proc return FALSE then define a space version of the proc that does the check then returns TRUE.

Never istype() src.